clientv3/client.go

```diff
@@ -463,6 +466,15 @@ func (c *Client) checkVersion() (err error) {
 // ActiveConnection returns the current in-use connection
 func (c *Client) ActiveConnection() *grpc.ClientConn { return c.conn }
 
+// CheckHealth checks the health status of gRPC server of current connection.
```

Comment:
don't expose through clientv3 yet? User can get this by `healthpb.NewHeathClient(c.ActiveConnection())` already